spring-boot-admin-client/pom.xml

```diff
@@ -60,6 +60,11 @@
             <artifactId>spring-webflux</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>io.projectreactor.netty</groupId>
+            <artifactId>reactor-netty-http</artifactId>
+            <optional>true</optional>
```

Comment:
I'm not sure how to define the dependency here correctly. 

webflux is optional and you only need the reactor-netty-http in webflux case, so this is optional, too. 

But when webflux is used, this is now required, right? So the users would need to include this dependency additionally? How do they know this?